collections/Cargo.toml

```
@@ -20,6 +20,8 @@ maintenance = { status = "experimental" }
 num-traits = "0.2"
 serde = "1.0"
 
+nimiq-serde = { path = "../serde" }
```

Use the workspace inheritance feature:
```suggestion
nimiq-serde = { workspace = true }
```